gimp-git/snapcraft.yaml

```diff
@@ -0,0 +1,221 @@
+name: gimp-edge
+version: 2.9.3~git
+#licence: $SNAP/gimp/src/LICENCE.txt
+summary: GNU Image Manipulation Program - www.gimp.org
+description: |
+    A freely distributed program for such tasks as photo retouching, image
+    composition and image authoring.  Whether you are a graphic designer,
+    photographer,  illustrator,  or scientist,  GIMP provides you with
+    sophisticated tools to get your job done.  You can further enhance your
+    productivity with GIMP thanks to many customization options and 3rd party
+    plugins.
+
+apps:
+    gimp:
+        command: gimp
+        plugs: [no-security]
+
+plugs:
+    no-security:
+        interface: old-security
+        security-template: unconfined
```

Comment:
this does not exist anymore.

Instead use no plugs at all in the `gimp` app in `apps` and add a toplevel key which has this `confinement: devmode`